virtcontainers/container.go

```diff
@@ -51,6 +51,10 @@ var cdromMajors = map[int64]string{
 	32: "CM206_CDROM_MAJOR",
 }
 
+// https://github.com/torvalds/linux/blob/master/include/uapi/linux/major.h
+// #FLOPPY_MAJOR		2
```

Comment:
For reference, according to that file two devices share this major number:

```c
#define FLOPPY_MAJOR		2
#define PTY_MASTER_MAJOR	2
```

Reply:
I suspect one is a block device (fd) and the other a char (pty), just for ref.

Reply:
yes, one is a block device while the other is a char one.